.travis.yml

```
@@ -31,6 +31,7 @@ matrix:
     - { <<: *linux, env: SKIP_TESTS=1 TARGET=mips64-unknown-linux-gnuabi64   }
     - { <<: *linux, env: SKIP_TESTS=1 TARGET=mips64el-unknown-linux-gnuabi64 }
     - { <<: *linux, env: SKIP_TESTS=1 TARGET=s390x-unknown-linux-gnu         }
+    - { <<: *linux, env: SKIP_TESTS=1 TARGET=x86_64-unknown-linux-musl       }
```

I was slightly confused at first by the `.travis.yml` file, but I think I understand what is meant.

While this PR has an inclusion to `.travis.yml` of a new TARGET to the matrix which _does_ get expanded with `if: branch != master`, for _PR builds_ on Travis, `branch` expands to the base branch for the PR.  Obviously, `master != master` is false, so this job isn't run and hence the PR is basically testing the functionality of what's on `master`.

So to actually test these changes prior to merging, the only change required would be to (temporarily) hand-code the matrix addition to always run? That might look something like:

```suggestion
    - { <<: *linux, env: SKIP_TESTS=1 TARGET=x86_64-unknown-linux-musl       }
    - { os: linux, env: SKIP_TESTS=1 TARGET=x86_64-unknown-linux-musl        }
```

(My suggestion would not be ideal to ship, but would illuminate any build failures.)